dashboard/functions/index.js

```diff
@@ -10,14 +10,19 @@ if (!process.env.GCLOUD_PROJECT) {
   process.env.GCLOUD_PROJECT = PROJECT_ID;
 }
 
+const UDMI_TARGET = process.env.UDMI_TARGET || 'udmi_target';
```

Comment:
I'm a bit concerned that we're ending up with two different "configuration" mechanisms here... one in the deploy_versions.js file, and the other through ENV variables. I'd be much happier if we could consolidate on one or the other. (I'm less concerned about the GCLOUD_PROJECT one since that's really outside UDMI itself).